VideoLocker/src/main/java/org/edx/mobile/user/DiscussionUser.java

```diff
@@ -0,0 +1,35 @@
+
+package org.edx.mobile.user;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class DiscussionUser implements Serializable {
+
+    @SerializedName("profile")
+    private Profile profile;
+
+    public Profile getProfile() {
+        return profile;
+    }
+
+    public void setProfile(Profile profile) {
```

Comment:
This setter doesn't seem to be useful, since the fields should be set only by the Gson parser. Same goes for the inner `Profile` class.